.vale/styles/Localization/AdjectivesWithNouns.yml

```diff
@@ -0,0 +1,11 @@
+---
+extends: sequence
+message: "Always use 'following' as an adjective before a noun for clarity and grammatical correctness. For example, use 'Complete the following steps' instead of 'Complete the following'."
```

Comment:
This is correct but I think it would be more concise to say:
"Do not use adjectives without a noun following them. For example, use 'Complete the following steps' instead of 'Complete the following'."

Reply:
Done! Updated as suggested!